packages/svelte2tsx/src/svelte2tsx.ts

```diff
@@ -401,6 +400,50 @@ function processInstanceScriptContent(str: MagicString, script: Node): InstanceS
         str.remove(exportStart, exportEnd);
     };
 
+    const propTypeAssertToUserDefined = (node: ts.VariableDeclarationList) => {
+        if (node.flags !== ts.NodeFlags.Let) {
+            return;
+        }
+
+        const hasInitializers = node.declarations.filter(
+            (declaration) => declaration.initializer
+        );
+        const handleTypeAssertion = (declaration: ts.VariableDeclaration) => {
+            const identifier = declaration.name;
+            const tsType = declaration.type;
+            const jsDocType = ts.getJSDocType(declaration);
+            const type = tsType || jsDocType;
+
+            if (!ts.isIdentifier(identifier) || !type) {
+                return;
+            }
+            const name = identifier.getText();
+            const end = declaration.end + astOffset;
+
+            str.appendLeft(end, `;name = __sveltets_any(${name});`);
```

Comment:
I guess you meant to write `${name} =` instead of `name =`? If I name my exports different than `name`, I get a strange error.